Stratis.Bitcoin/Configuration/TextFileConfiguration.cs

```diff
@@ -151,10 +148,13 @@ public bool Contains(string key)
         public string[] GetAll(string key)
         {
             List<string> values;
-            if (!this.args.TryGetValue(key, out values))
-                if (!this.args.TryGetValue($"-{key}", out values))
-                    return new string[0];
-            return values.ToArray();
+            List<string> dashValues;
+            // get the values without the - prefix
+            this.args.TryGetValue(key, out values);
+            // get the values with the - prefix
+            this.args.TryGetValue($"-{key}", out dashValues);
+            // concat those
+            return (values ?? new List<string>()).Concat(dashValues ?? new List<string>()).ToArray();
```

Comment:
This construction is unnecessary. It requires too much brainpower to process.

Consider this instead:
```
// Get the values from argument without '-' prefix.
if (!this.args.TryGetValue(key, out values))
    values = new List<string>();

// Get the values from argument with '-' prefix.
if (!this.args.TryGetValue($"-{key}", out dashValues))
    dashValues = new List<string>();

return values.Concat(dashValues).ToArray();
```

I also don't need the trivial "concat those" comment because the return line is readable and it is obvious.